examples/oneapi/cpp/source/knn/knn_kd_tree_batch.cpp

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
```

Comment:
Rename example `knn_kd_tree_batch.cpp` -> `knn_cls_kd_tree_dense_batch.cpp`. Algorithm - Type of task - Method - Input format - Mode